src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -79,7 +79,11 @@ databaseChangeLog:
   - include:
       file: changesets/changelog_2022-08-23T14:02:44Z.xml
       relativeToChangelogFile: true
-
+      
```

Comment:
You added some blank spaces on this line

Reply:
fixed